cds_books/literature/covers.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS Books is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Literature covers."""
+
+
+def build_literature_cover_urls(metadata):
+    """Decorate literature with cover urls for all sizes."""
+    client = "cernlibrary"
```

Comment:
I would say that we should have this is a config/secret... it is not a secret because it is in the URL, but it is kind of bothering me! WDYT?

Reply:
👍to that